circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/replica/Replica.java

```diff
@@ -426,4 +436,25 @@ public TableAndStatistics getTableAndStatistics(TableReplication tableReplicatio
     return super.getTableAndStatistics(tableReplication.getReplicaDatabaseName(),
         tableReplication.getReplicaTableName());
   }
+
+  private void dropReplicaTable(CloseableMetaStoreClient client, String replicaDatabaseName, String replicaTableName) {
+    LOG.debug("Replication mode: FULL_OVERWRITE. Dropping existing replica table.");
+    try {
+      if (client.tableExists(replicaDatabaseName, replicaTableName)) {
```

Comment:
This call is a "new" addition of the hive API I'm not sure if we use it anywhere else but if we do it is ok but if we don't better to avoid using it as it's not always there on old client.
Better to just do the 
```
Try
dropTable() 
Catch(NoSuchObjectException e) {
//nothing to drop table doesn't exist etc.....
}
```